packages/runtime/src/components/builtin/Text/Text.tsx

```diff
@@ -16,7 +13,7 @@ type Props = {
   margin?: string
 }
 
-const Text = forwardRef(function Text(props: Props, ref: ForwardedRef<unknown>) {
+const Text = forwardRef(function Text(props: Props, ref: ForwardedRef<any>) {
```

Comment:
I want to make sure I don't run into any weird issues with lazyloading while I work on this feature. So I am no using next/dynamic and this any type is a temporary artifact of that.